web/pages/database/[databaseName]/data/[refName]/[tableName].tsx

```diff
@@ -6,8 +6,8 @@ import { GetServerSideProps, NextPage } from "next";
 type Props = {
   params: TableParams & {
     active?: string;
+    edit?: boolean;
```

Comment:
`edit` needs to be explicitly passed to `DatabasePage.ForTable` now that it isn't a `prop`